libs/common/src/sql-db-entities/fee-record/fee-record.entity.test.ts

```diff
@@ -79,4 +79,29 @@ describe('FeeRecordEntity', () => {
       return UtilisationReportEntityMockBuilder.forStatus('PENDING_RECONCILIATION').build();
     }
   });
+
+  describe('removeAllPayments', () => {
+    it("removes all payments, sets the report status to 'TO_DO' and updates the 'lastUpdatedBy...' fields", () => {
+      // Arrange
+      const feeRecord = FeeRecordEntityMockBuilder.forReport(aUtilisationReport()).build();
```

Comment:
About to push a change to set a different status and assign payments to this fee record